src/components-examples/material/form-field/form-field-custom-control/example-tel-input-example.html

```diff
@@ -1,7 +1,8 @@
 <div [formGroup]="parts" class="example-tel-input-container">
-  <input class="example-tel-input-element" formControlName="area" size="3" aria-label="Area code" (input)="_handleInput()">
+  <input class="example-tel-input-element" formControlName="area" size="3" 
+  maxLength="3" aria-label="Area code" (input)="_handleInput(parts.controls.area,exchange)"  #area>
   <span class="example-tel-input-spacer">&ndash;</span>
-  <input class="example-tel-input-element" formControlName="exchange" size="3" aria-label="Exchange code" (input)="_handleInput()">
+  <input class="example-tel-input-element" formControlName="exchange" maxLength="3" size="3" aria-label="Exchange code" (input)="_handleInput(parts.controls.exchange,subscriber)" (keyup.backspace)="autoFocusPrev(parts.controls.exchange,area)" #exchange>
   <span class="example-tel-input-spacer">&ndash;</span>
-  <input class="example-tel-input-element" formControlName="subscriber" size="4" aria-label="Subscriber number" (input)="_handleInput()">
+  <input class="example-tel-input-element" formControlName="subscriber" maxLength="4" size="4" aria-label="Subscriber number" (input)="_handleInput(parts.controls.subscriber,subscriber)" (keyup.backspace)="autoFocusPrev(parts.controls.subscriber,exchange)" #subscriber>
```

Comment:
`_handleInput` doesn't need a second argument in this case

Reply:
resolved